### PR TITLE
Chart of the number of products sold by each brand

### PR DESCRIPTION
**Pull request by NgocAnhDT**
## Related Tickets:
- [#48037](https://edu-redmine.sun-asterisk.vn/projects/hn_oe49-shoe-store/issues/48037)

## WHAT this PR does?
Chart of the number of products sold by each brand

## Notes Impacted Areas:
- N/A.

## Attachments:
Screenshot from 2022-04-07 17-02-36

### CODE REVIEW

**Review by ariesh2t — APPROVED** [review]

**Comment by NgocAnhDT** [comment]
ready

File: resources/views/admins/dashboard.blade.php [hunk 1]
```diff
@@ -7,4 +7,29 @@
...
+                    $selected_day = '';
+                    $selected_month = '';
+                    if (request()->filter == 'day') {
+                        $selected_day = 'selected';
+                    } else {
+                        $selected_month = 'selected';
+                    }
+                @endphp
+                <option value="day" {{ $selected_day }}>{{ __('7 days ago') }}</option>
+                <option value="month" {{ $selected_month }}>{{ __('sub month') }}</option>
```
Comment (huyenttm-0901) [inline resources/views/admins/dashboard.blade.php:24]: viết if vào trong option chứ em
<option value="day" @if (request()->filter == 'day') selected @endif>{{ __('7 days ago') }}</option

**Review by huyenttm-0901 — COMMENTED** [review]
theem attachment vaof PR description

**Comment by NgocAnhDT** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**